packages/novu/README.MD

```diff
@@ -31,13 +31,31 @@
 ```bash
 npx novu@latest dev
 ```
-  
+
+## 🔥 Flags
+
+| flag | long form usage example | description                   | default value               |
+| ---- | ----------------------- | ----------------------------- | --------------------------- |
+| -p   | --port <port>           | local bridge application port | 4000                        |
+| -r   | --route <route>         | bridge application route      | /api/novu                   |
+| -o   | --origin <origin>       | bridge application origin     | http://localhost            |
+| -d   | --dashboard-url <url>   | Novu cloud dashboard URL      | https://dashboard.novu.co   |
+| -sp  | --studio-port <port>    | Local Studio server port      | 2022                        |
+| -t   | --tunnel <url>          | self hosted tunnel url        | https://my-tunnel.ngrok.app |
```

Comment:
@jainpawan21 , could we update these docs to align with the change in [this Docs PR](https://github.com/novuhq/docs/pull/722/files#diff-e73937ecbcc0744e250ab8ba1a4db43570a4e067077912e654ee73c6dea2d996R43)?